### PR TITLE
Add social media brand colors pack (Closes #471)

### LINKED ISSUE #471: Add social media brand colors pack
```css
:where(html) {
  --facebook: #1877f2;
  --facebook-old: #3b5998;
  --twitter: #1da1f2;
  --youtube: #ff0000;
  --instagram: #c32aa3;
  --instagram-blue: #4c5fd7;
  --instagram-purple: #7232bd;
  --instagram-orange: #f46f30;
  --instagram-yellow: #ffdc7d;
  --google-blue: #4285f4;
  --google-red: #ea4335;
  --google-yellow: #fbbc05;
  --google-green: #34a853;
  --pinterest: #bd081c;
  --snapchat: #fffc00;
  --whatsapp: #25d366;
  --whatsapp-teal: #075e54;
  --whatsapp-teal2: #128c7e;
  --tiktok-black: #010101;
  --tiktok-blue: #69c9d0;
  --tiktok-pink: #ee1d52;
  --tiktok-white: #fff;
  --mastodon: #563acc;
  --apple: #a6b1b7;
  --amazon: #ff9900;
  --alexa: #00a7ce;
  --microsoft-red: #f35022;
  --microsoft-green: #80bb03;
  --microsoft-blue: #03a5f0;
  --microsoft-yellow: #ffb903;
  --periscope: #40a4c4;
  --foursquare-pink: #f94877;
  --foursquare-navy: #073282;
  --foursquare-blue: #2d5be3;
  --yelp: #d32323;
  --swarm: #ffa633;
  --medium: #02b875;
  --skype: #00aff0;
  --skype-dark-blue: #0078d7;
  --android: #a4c639;
  --stumbleupon: #e94826;
  --flickr-pink: #f40083;
  --flickr-blue: #006add;
  --yahoo: #430297;
  --twitch: #9146ff;
  --soundcloud: #ff5500;
  --spotify: #1ed760;
  --spotify-dark-green: #1db954;
  --dribbble: #ea4c89;
  --slack-purple: #4a154b;
  --slack-blue: #36c5f0;
  --slack-green: #2eb67d;
  --slack-red: #e01e5a;
  --slack-yellow: #ecb22e;
  --reddit: #ff5700;
  --deviantart: #05cc47;
  --pocket: #ee4056;
  --quora: #2b6dad;
  --slideshare-orange: #e68523;
  --slideshare-blue: #00a0dc;
  --fivehundredpx: #0099e5;
  --vk: #4a76a8;
  --listly-orange: #df6d46;
  --listly-blue: #52b1b3;
  --vine: #00b489;
  --steam: #171a21;
  --discord: #5865f2;
  --telegram: #0088cc;
}
```

this was generated by Gemini.. needs reviewed and pruned (like deviantart, vine, etc.. we dont need those right?). Also what's missing? Bluesky?

### PR DESCRIPTION
**Pull request by aspizu**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -44,6 +44,7 @@
     "./src/zindex": "./src/props.zindex.js",
     "./src/masks.edges": "./src/props.masks.edges.js",
     "./src/masks.corner-cuts": "./src/props.masks.corner-cuts.js",
+    "./src/brand": "./src/props.brand.js",
```
Comment (argyleink) [inline package.json:47]: add another entry into this file that points to the css file, somewhere next to the other colors 

something like "./brand": "path-to-brand-css-file",

File: src/props.brand.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+  // source: https://cdn-dynmedia-1.microsoft.com/is/content/microsoftcorp/microsoft/mscle/documents/presentations/CELA_ThirdPartyLogoGuidelines_June2021.pdf
+  '--brand-microsoft-red': '#F25022',
+  '--brand-microsoft-green': '#7FBA00',
+  '--brand-microsoft-blue': '#00A4EF',
+  '--brand-microsoft-yellow': '#FFB900',
+  // source: https://usbrandcolors.com/pinterest-colors/
+  '--brand-pinterest': '#E60023',
+  // source: https://developer.spotify.com/documentation/design#using-our-colors
+  '--brand-spotify': '#1ED760',
+  // source: https://discord.com/branding
```
Comment (argyleink) [inline src/props.brand.js:34]: adding the sources in here was a great idea 👍🏻

**Review by argyleink — CHANGES_REQUESTED** [review]
overall looks great!
missing 1 entry in the exports of package.json, then that wraps up the props work

are you open to adding a section to the docsite that shows all the colors?

File: docsite/js/brand-colors.js [hunk 3]
```diff
@@ -0,0 +1,8 @@
+import brandProps from '../../src/props.brand-colors.js'
+
+const brandColors = document.querySelector('#brand-colors')
+
+for (const [brand, color] of Object.entries(brandProps)) {
+  const brandTitlecase = brand.replace(/-/g, ' ').replace('brand ', '')
+  brandColors.innerHTML += `<ul class="open-colors count-em color-swatch-list"><h4>${brandTitlecase}</h4><li><button class="color-swatch" style="background-color: var(${brand})" aria-label="${brandTitlecase}"></button></li></ul>`
```
Comment (argyleink) [inline docsite/js/brand-colors.js:7]: this happens on the client and not build time yeah?
can you take the output of it and drop the markup in the build, so there's no JS required to render them?

File: docsite/index.js [hunk 4]
```diff
@@ -1,10 +1,10 @@
...
-import '/js/slider.js'
+import '/js/brand-colors.js'
+import '/js/color-copy.js'
+import '/js/easing.js'
 import '/js/scrollspy.js'
 import '/js/select.js'
-
-import '/js/color-copy.js'
\ No newline at end of file
+import '/js/slider.js'
```
Comment (argyleink) [inline docsite/index.js:10]: looks like a linter ran here? if nothing changed, can you remove it from the PR? 🙏🏻

File: docsite/index.html [hunk 5]
```diff
@@ -526,6 +526,11 @@ <h4>File Sizes</h4>
               <div class="file">
                 <a href="https://github.com/argyleink/open-props/blob/main/src/props.zindex.css">zindex.css</a>
               </div>
+
+              <div class="size"><strong>0.30</strong>kB</div>
+              <div class="file">
+                <a href="https://github.com/argyleink/open-props/blob/main/src/props.brand-colors.css">brand-colors.css</a> <a href="https://codepen.io/aspizu/pen/LYwNeyz"><small class="green-badge">demo</small></a>
+              </div>
```
Comment (argyleink) [inline docsite/index.html:533]: rad, thanks for noticing this and contributing it 🤘🏻

**Review by argyleink — CHANGES_REQUESTED** [review]

File: docsite/index.js (hunk 4 above)
Comment (aspizu) [reply docsite/index.js]: is it fine now?

File: docsite/js/brand-colors.js (hunk 3 above)
Comment (aspizu) [reply docsite/js/brand-colors.js:7]: maybe we should switch to astro or something.

Comment (argyleink) [reply docsite/js/brand-colors.js:7]: indeed, the next version of the docs is using astro, but for now, we're hand rollin

**Review by argyleink — APPROVED** [review]
awesome, thanks for pushing the prerendered markup and waiting for me while I was on bereavement.

**Comment by argyleink** [comment]
available in `v1.7.7`, site update coming soon

**Comment by aspizu** [comment]
I would appreciate it if you could add the `hacktoberfest-accepted` label to my PR.

**Comment by aspizu** [comment]
thank you!

**Pull request merged**